articles/lets-learn-about-osi-layer-model/index.md

```diff
@@ -0,0 +1,133 @@
+
+### Advantages of the OSI Model
+The OSI model helps users and operators of computer networks:
+- Determine the required hardware and software to build their network.
+- Understand and communicate the process followed by components communicating across a network. 
+- Perform troubleshooting, by identifying which network layer is causing an issue and focusing efforts on that layer.
+
+The OSI model helps network device manufacturers and networking software, vendors:
+- Create devices and software that can communicate with products from any other vendor, allowing open interoperability.
+- Define which parts of the network their products should work with.
+- Communicate to users at which network layers their product operates 
+
+### Characteristics of OSI Model:
+
+OSI consists of several layers and each layer performs a particular network function. It's divided into two layers: upper layers and lower layers.
+
+The upper layer of the OSI model mainly deals with the application related issues, and they are implemented only in the software. The application layer is closest to the end-user. Both the end-user and the application layer interact with the software applications. An upper layer refers to the layer just above another layer.
+
+The lower layer of the OSI model deals with data transport issues. The data link layer and the physical layer are implemented in hardware and software. The physical layer is the lowest layer of the OSI model and is closest to the physical medium. The physical layer is mainly responsible for placing the information on the physical medium.
+
+OSI follows a vertical approach. ”If you can understand the OSI model and its layers, you can also then understand which protocols and devices can interoperate with each other when new technologies are developed and explained.
+
+- A layer should only be created where definite levels of abstraction are needed.
+- The function of each layer should be selected as per the internationally standardized protocols.
+- The number of layers should be large so that separate functions should not be put in the same layer. At the same time, it should be small enough so that architecture doesn't become very complicated.
+- In the OSI model, each layer relies on the next lower layer to perform primitive functions. Every level should able to provide services to the next higher layer.
+- Changes made in one layer should not need changes in other lavers.
+ 
+As mention, the model uses layers to help give a visual description of what is going on with a particular networking system. This can help network managers narrow. 
+
+### Functions of the OSI Layers
```

Comment:
Include the image for OSI model here.

Reply:
@lalith1403  can you please review I have included the image